### DISCUSSION TITLE
Support for prefers-color-scheme media query

### DISCUSSION THREAD

**Discussion by TeddyDD** [wailsapp/wails#2973] [Q&A]
I've been testing Wails v2 on Linux (webkitgtk), it seems it does not support dark theme media query

```css
@media (prefers-color-scheme: dark) {
...
}
```

It this feature supported on other platforms? Are there plans to add support for it on Linux?

**Comment by leaanthony** [comment]
This provides more context: https://bugs.chromium.org/p/chromium/issues/detail?id=998903
